lib/generators/rspec/controller/controller_generator.rb

```diff
@@ -7,8 +7,9 @@ class ControllerGenerator < Base
       argument :actions, :type => :array, :default => [], :banner => "action action"
 
       class_option :template_engine, :desc => "Template engine to generate view files"
-      class_option :controller_specs, :type => :boolean, :default => true
-      class_option :view_specs,       :type => :boolean, :default => true
+      class_option :controller_specs, :type => :boolean, :default => true,  :desc => "Generate controller specs"
+      class_option :view_specs,       :type => :boolean, :default => true,  :desc => "Generate view specs"
+      class_option :routing_specs,    :type => :boolean, :default => false,  :desc => "Generate routing specs"
```

Comment:
Double spacing here, unless you were intending to align the `:desc`, in which case the latst needs a space removed.